samza-core/src/main/java/org/apache/samza/clustermanager/StandbyContainerManager.java

```diff
@@ -126,6 +126,30 @@ public void handleContainerLaunchFail(String containerID, String resourceID,
     }
   }
 
+  /**
+   *  Handle the failed stop for a container, based on
+   *  Case 1. If it is standby container, continue the failover
+   *  Case 2. If it is an active container, then log a warning and return.
```

Comment:
Please correct the comment here, Case 2. If this is an active container, throw an exception restart the AM.